.github/workflows/pull-request-checks.yml

```diff
@@ -12,9 +12,34 @@ concurrency:
 
 permissions:
     contents: read
+env:
+    CI_LABEL: "ci:approve-public-fork-ci"
 
 jobs:
+    check-run-approval:
```

Comment:
Should check-run-approval be included in "pull request checks" step at the end?

Reply:
No for the same reason `affected-packages` isn't included.  This job always runs first and just blocks everything when it fails.